.openpublishing.redirection.json

```diff
@@ -919,6 +919,18 @@
             "source_path": "docs/fsharp/tutorials/getting-started/getting-started-command-line.md",
             "redirect_url": "/dotnet/fsharp/get-started/get-started-command-line",
             "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/csharp/getting-started/library-with-visual-studio.md",
```

Comment:
nit: I usually put these in alphabetical order so it's easier to find them if needed